x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/execution/ExecutionService.java

```diff
@@ -433,7 +433,14 @@ private void executeAsync(WatchExecutionContext ctx, final TriggeredWatch trigge
                             "Error storing watch history record for watch [{}] after thread pool rejection",
                             triggeredWatch.id()), exc);
                 }
-                deleteTrigger(triggeredWatch.id());
+                try {
+                    deleteTrigger(triggeredWatch.id());
+                } catch (Exception exc2) {
```

Comment:
any reason you named this exc2? Good catch regardless.

Reply:
the one above it is `exc`